core/src/network/event.rs

```diff
@@ -20,28 +20,24 @@
 
 //! Network events and associated information.
 
-use crate::{
-    Multiaddr,
-    connection::{
-        ConnectionId,
-        ConnectedPoint,
-        ConnectionError,
-        ConnectionHandler,
-        Connected,
-        EstablishedConnection,
-        IntoConnectionHandler,
-        ListenerId,
-        PendingConnectionError,
-    },
-    transport::Transport,
-};
+use crate::{Multiaddr, connection::{
+    ConnectionId,
+    ConnectedPoint,
+    ConnectionError,
+    ConnectionHandler,
+    Connected,
+    EstablishedConnection,
+    IntoConnectionHandler,
+    ListenerId,
+    PendingConnectionError,
+}, transport::Transport, PeerId};
```

Comment:
I don't usually comment on code formatting since I can work with almost anything, but this particular mixture of line breaks and no line breaks looks like its unduly giving prominence to the indented block of imports, with the others squashed between the opening and closing braces. Was this done automatically or manually? At the end of the day I'm sure I can work with this as well, I'm just curious.

Reply:
I tried to not make any formatting changes but this one must have slipped through! Was definitely not on purpose but probably auto-formatted by my IDE :grimacing:

Reply:
Reverted the formatting changes :)